CHANGELOG.md

```diff
@@ -35,7 +44,25 @@ The format is based on [Keep a Changelog], and this project adheres to
 
 ### Fixed
 
-- Fixed scenarios not rendering with Rails 6 ([#93]).
+- Fixed scenarios not rendering with Rails 6 ([#93]). Please ensure formats are
+  specified using symbols when defining structures:
+
+  ```diff
+   def formats
+  -  ['html', 'text']
+  +  [:html, :text]
+   end
+
+   def layout_templates
+     {
+  -    'html' => 'layouts/my_html_layout',
+  -    'text' => 'layouts/my_text_layout'
+  +    html: 'layouts/my_html_layout',
+  +    text: 'layouts/my_text_layout'
+     }
+   end
+  ```
```

Comment:
An additional consequence of the fix in #93: downstream users will need to ensure formats are specified as symbols, when defining structures.